pkg/controllers/dynakube/injection/conditions.go

```diff
@@ -0,0 +1,34 @@
+package injection
+
+import (
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	metaDataEnrichmentConditionType   = "MetadataEnrichment"
+	codeModulesInjectionConditionType = "CodeModulesInjection"
+
+	nsAndSecretsCreatedReason  = "NamespaceAndSecretsCreated"
+	nsAndSecretsCreatedMessage = "Namespaces mapped and secrets created"
```

Comment:
do we have to be this specific? (mainly in the `Reason`)

Reply:
Is it even correct that the namespace is created alongside the secret?

Reply:
true, it would be more correct to say `NamespacesMappedAndSecretsCreated`, but pls no 😅

Reply:
Yeah I agree that it is to specific anyways 😅 

Probably `SecretCreated` is the way to go I suppose